inst/rmarkdown/templates/trundler_template/skeleton/skeleton.Rmd

```diff
@@ -13,7 +13,7 @@ output: html_document
 knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE, fig.width = 10, comment = NULL)
 ```
 
-```{r libraries}
+```{r libraries and constants}
```

Comment:
Probably best to hyphenate the chunk label like`libraries-and-constants`. But, at the same time, not all chunks need a label and this one could probably go without.